core/procedures.js

```diff
@@ -92,6 +92,31 @@ Blockly.Procedures.allProcedureMutations = function(root) {
   return mutations;
 };
 
+/**
+ * Sorts an array of procedure definition mutations alphabetically.
+ * (Does not mutate the given array.)
+ * @param {!Array.<Element>} mutations Array of mutation xml elements.
+ * @return {!Array.<Element>} Sorted array of mutation xml elements.
+ */
+Blockly.Procedures.sortProcedureMutations = function(mutations) {
```

Comment:
This only needs to be used inside scratch blocks, so let's make it private: add an underscore at the end of the name, and add @private below the @return annotation in the comment.

Reply:
Fixed in https://github.com/LLK/scratch-blocks/pull/1254/commits/944ac2ce3e3b63d54b3cbd33cc1704b60b77a114 / https://github.com/LLK/scratch-blocks/pull/1254/commits/abcb8388a550ce61749fadff4a744b4a40e383fc.